pkg/osbuild/ostree_deploy_container_stage.go

```diff
@@ -0,0 +1,67 @@
+package osbuild
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const ostreeContainerTargetImgrefRegex = "^(ostree-remote-registry|ostree-image-signed|ostree-unverified-registry):.*$"
+
+// Options for the org.osbuild.ostree.deploy.container stage.
+type OSTreeDeployContainerStageOptions struct {
+
+	// Name of the stateroot to be used in the deployment
+	OsName string `json:"osname"`
+
+	// Additional kernel command line options
+	KernelOpts []string `json:"kernel_opts,omitempty"`
+
+	// Image ref used as the source of truth for updates
+	TargetImgref string `json:"target_imgref"`
```

Comment:
(super-nitpick) I personalyl like `TargetImageRef` but that is probably the mother of all bikesheding (sorry!)

Reply:
Aha, sure - this defined outside? that makes sense and then yes, it should be as close as possible.

Reply:
> this defined outside?

Yup, it gets serialised into the manifest for the stage options.